vs_utils/scripts/public_data/vina_prep.py

```diff
@@ -0,0 +1,60 @@
+"""
+Ligand and receptor preparation for AutoDock/Vina.
+"""
+import argparse
+import os
+import subprocess
+import tempfile
+
+
+def parse_args(input_args=None):
+  """
+  Parse command-line arguments.
+
+  Parameters
+  ----------
+  input_args : list, optional
+    Input arguments. If not provided, defaults to sys.argv[1:].
+  """
+  parser = argparse.ArgumentParser()
+  parser.add_argument('input',
+                      help='Input filename.')
+  parser.add_argument('-r', '--receptor', action='store_true',
+                      help='Treat input as a receptor.')
+  parser.add_argument('-s', '--split', action='store_true',
+                      help='Split output files into one file per molecule.')
+  return parser.parse_args(input_args)
+
+
+def main(input_filename, receptor=False, split=False):
+  output_filename = os.path.basename(input_filename)
```

Comment:
Would it be possible to factor these changes into a class? For my particular use case, I can't make use of subprocess when running python. If a class is tricky, a neat shell-script would actually be a good alternative as well.